.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: Release sanity
+
+on:
+  pull_request:
+    branches: [master]
+    types: [labeled]
+
+jobs:
+  ci-release-build:
+    name: "Sanity check release commits"
+    if: ${{ github.event.label.name == 'kind/release' }}
```

Comment:
Is this a label a human would add? Is it possible to just make it queue it off of the commit title being "Release ..." instead?

Reply:
Yes, it would need to be manually added. I'll check to see if the latter is possible.

Reply:
Hum...configuring which CI to run via labels sounds really nice as a general pattern.  For example it'd be nice to label things "ci/trivial" to skip the expensive tests.  But we'd need to plumb that through various systems to make it work.

Reply:
@jlebon I've scouted GH docs a bit, and I think I found all the primitives I need to put that together. I'll try gluing them together in a separate repo and report back if it works.

Reply:
Having it be a manual step WFM as long as we document it somewhere (hmm, I think we should have something like https://github.com/coreos/rpm-ostree/blob/master/docs/RELEASE.md for ostree?)